model/feed.go

```diff
@@ -25,3 +26,17 @@ type Feed struct {
 	Avatar   string `json:"author_avatar,omitempty" meddler:"build_avatar,zeroisnull"`
 	Email    string `json:"author_email,omitempty"  meddler:"build_email,zeroisnull"`
 }
+
+type Feeds []*Feed
+
```

Comment:
can we keep this type in the `store/datastore` package since it probably only used internally in that package?